Sprint 11

Maksim-Isaev opened the pull request:
Задание спринта 11 выполнено! 

Для разработки использовал Beta Android Studio Jellyfish | 2023.3.1 RC 2
Build #AI-233.14808.21.2331.11678080, built on April 5, 2024

Maksim-Isaev committed:
Sprint 11. HW.
1. Добавил зависимости  для Retrofit  в build.gradle
2. Переименован значок плейсхолдера для иконок
3. Начато добавление фичей согласно дз.

Maksim-Isaev committed:
Sprint 11. HW.
1. Обновлен activity_search.xml под шаблон

Maksim-Isaev committed:
Sprint 11. HW.
1. Обновлен activity_search.xml под шаблон

Maksim-Isaev committed:
Sprint 11. HW.
1. Добавлееы классы для работы API (api.kt, ItunesResponse.kt, ResultResponse.kt)
2. Скорректирован класс Track.kt для корректной работы api
3. TrackAdapter.kt - Адаптер отображения списка треков скорректирован, добавлен список для отображения в RecyclerView
4. TrackViewHolder.kt - скорректировано форматирование и установка времени трека
5. Добавлены комментарии по коду

Maksim-Isaev committed:
Sprint 11. HW.
1. Обновлен SearchActivity.kt для работы с api - api пока не работает - ищу проблему
2. обновлен activity_search.xml так как в textview сидела ошибка которая приводила к вылету приложения при открытии окна поиск

Maksim-Isaev committed:
Sprint 11. HW.
1. Решена проблема с неработающим поиском . - Поиск песен работает.
Задание 11 спринта выполнено в полном обьеме

DmitrijEgorow commented:
> Задание спринта 11 выполнено!
> 
> Для разработки использовал Beta Android Studio Jellyfish | 2023.3.1 RC 2 Build #AI-233.14808.21.2331.11678080, built on April 5, 2024

Лучше agp версии не rc

DmitrijEgorow commented:
https://github.com/Maksim-Isaev/Playlist-Maker/assets/36837067/313fec2d-81c5-411c-b859-fd03d8d33488

Крестик не работает

DmitrijEgorow commented:
![Screenshot_20240527_125159](https://github.com/Maksim-Isaev/Playlist-Maker/assets/36837067/f057ac7f-a8b2-4afc-911e-1a9b2c7429d4)
Кнопка не соответствует дизайну

DmitrijEgorow commented:
![Screenshot_20240527_135340](https://github.com/Maksim-Isaev/Playlist-Maker/assets/36837067/c0705058-8125-4c19-9237-84f9ac32a381)
Разъезжается длительность

Maksim-Isaev committed:
дз 11 (переехал на мак) - кнопка отмены заработала снова но она сьехала вправо - решаю

Maksim-Isaev committed:
Sprint 11. HW.
Исправлены ошибки после ревью:
1- активированна кнопка отмены в поиске,
2 - поправлена кнопка обновить

Maksim-Isaev commented:
Скорректировал код согласно замечаниям

Maksim-Isaev committed:
поправка

Maksim-Isaev commented:
Поправил, видимо глюкнула студия и удалила строки в импорте. Проверил на телефоне  - запустилось


![photo_5388895376553139865_y](https://github.com/Maksim-Isaev/Playlist-Maker/assets/152393297/594efb1c-6a3d-4d86-addb-5b9d82c32e99)
